agent/pkg/builder/container/container_builder.go

```diff
@@ -28,6 +29,7 @@ type ContainerBuilder interface {
 	WithPortRange(portRanges []PortRange) ContainerBuilder
 	WithMountPoints(mounts []mount.Mount) ContainerBuilder
 	WithName(name string) ContainerBuilder
+	WithNetworkAliases(aliases []string) ContainerBuilder
```

Comment:
Variadic parameters would also work, work better, consuming arrays and multiple parameters the same way.